dynamic-config/config-cc/src/main/java/org/apache/servicecomb/config/archaius/sources/ConfigCenterConfigurationSourceImpl.java

```diff
@@ -126,7 +128,23 @@ public void handle(String action, Map<String, Object> parseConfigs) {
       if (parseConfigs == null || parseConfigs.isEmpty()) {
         return;
       }
+
       Map<String, Object> configuration = ConfigMapping.getConvertedMap(parseConfigs);
+      List<String> fileSourceList = new ArrayList<>();
+	  
+	  try {
+		  fileSourceList = configCenterClient.getFileSources();
+	  } catch (NullPointerException e) {
```

Comment:
1. use IDE formatter in etc folder to format code
2. do not catch NullPointerException, figure out when NullPointerException is thrown, and add null check .